command/genesis/params.go

```diff
@@ -227,6 +224,12 @@ func (p *genesisParams) initGenesisConfig() error {
 		Bootnodes: p.bootnodes,
 	}
 
+	if p.parsedBaseFeeConfig != nil {
+		chainConfig.Genesis.BaseFee = p.parsedBaseFeeConfig.baseFee
+		chainConfig.Genesis.BaseFeeChangeDenom = p.parsedBaseFeeConfig.baseFeeChangeDenom
+		chainConfig.Genesis.BaseFeeEM = p.parsedBaseFeeConfig.baseFeeEM
```

Comment:
@goran-ethernal and myself were discussing about moving these parameters into `chain.Params` struct. Just be mindful to populate `BaseFee` to the `Genesis` as well (we need it there by specification).

Reply:
I've moved the `BaseFeeChangeDenom` to `chain.Params` struct in the governance PR, since it is part of on chain governance, and can be changed in runtime. But, if this gets merged after governance, it will require a change, but it shouldn't be a problem.

Reply:
Ok, as agreed I'll address my point through governance PR.